Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/model/DccWalletInfoOutput.kt

```diff
@@ -280,7 +320,16 @@ data class ReissuanceDivision(
     val longText: CclText?,
 
     @JsonProperty("faqAnchor")
-    val faqAnchor: String?
+    val faqAnchor: String?,
+
+    @JsonProperty("identifier")
+    val identifier: String? = "renew",
```

Comment:
It is, it is supposed to be set to renew in case the wallet info is calculated with v2, but it is possible to overwrite with null. It is only relevant for notifications which have not been adjusted yet